sys/linux/test/bpf_cgroup

```diff
@@ -0,0 +1,25 @@
+# First, write to an entry of /proc/sys. It should work fine.
+
+r0 = openat$tcp_congestion(AUTO, &AUTO='/proc/sys/net/ipv4/tcp_congestion_control', AUTO, AUTO)
+
+r1 = write$tcp_congestion(r0, &AUTO='reno\x00', AUTO)
+
+# Now, load a BPF_PROG_TYPE_CGROUP_SYSCTL that simply returns 0, which will block all writes to /proc/sys
+
+r2 = bpf$PROG_LOAD(AUTO, &AUTO={0x17, AUTO, &AUTO=@framed={{AUTO, AUTO, AUTO, AUTO, 0x0, AUTO, AUTO, AUTO, 0x0}, [], {AUTO, AUTO, AUTO, AUTO}}, &AUTO='GPL\x00', 0x0, 0x0, 0x0, 0x0, 0x0, [0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0], 0x0, 0x12, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0}, 0x78)
+
+r3 = openat(0xffffffffffffff9c, &AUTO='./cgroup\x00', 0x0, 0x0)
```

Comment:
I tried `./cgroup/syz0` but it did not work (failed with `errno = 2 [No such file or directory]`). Not sure what is happening

Reply:
Does syz-execprog -debug print anything about cgroups?
If it does not give any glues, I would suggest to add more logging to syz-executor around cgroups calls to see what fails.
It's possible something broke, or there are some implicit assumptions about the image. This part is not well tested.

Reply:
It seems like `./cgroup` is the correct directory, as we mount it [here](https://github.com/google/syzkaller/blob/20366b870db78f1d58502dad9ffcf6dad618fae7/executor/common_linux.h#L3496).
But then I wonder if [this](https://github.com/google/syzkaller/blob/20366b870db78f1d58502dad9ffcf6dad618fae7/sys/linux/cgroup.txt#L39) is correct.

Reply:
You mean the subdirectories?
The intention is that the test programs can create these subdirs (e.g. use in mkdirat). And the subdirs in cgroup directory mean nested cgroups.

Reply:
Got it. I think every thing is fine then.

Reply:
Thanks for double-checking